jwql/website/apps/jwql/data_containers.py

```diff
@@ -65,6 +68,36 @@
 REPO_DIR = os.path.split(PACKAGE_DIR)[0]
 
 
+def get_table_view_components(request):
```

Comment:
Could you move this function down in the module so that it is in alphabetical order?